src/eShop.AppHost/Extensions.cs

```diff
@@ -75,4 +75,28 @@ public static IDistributedApplicationBuilder AddOpenAI(this IDistributedApplicat
 
         return builder;
     }
+
+    /// <summary>
+    /// Configures eShop projects to use Ollama for text embedding and chat.
+    /// </summary>
+    public static IDistributedApplicationBuilder AddOllama(this IDistributedApplicationBuilder builder,
+        IResourceBuilder<ProjectResource> catalogApi,
+        IResourceBuilder<ProjectResource> webApp)
+    {
+        var ollama = builder.AddOllama("ollama")
+            .WithDataVolume()
+            .WithGPUSupport()
+            .WithOpenWebUI();
```

Comment:
How long does it take for the Ollama container to spin up? Should this have persistent lifetime?

Reply:
Ollama itself is a pretty lightweight container and starts fast (I have it running on a NUC), so I wouldn't do a persistent lifetime. The expensive part is model downloading, which is why I added the `WithDataVolume` so that you only download the models on first run.